<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QnAggregationWidget</class>
 <widget class="QWidget" name="QnAggregationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>38</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QCheckBox" name="enabledCheckBox">
     <property name="text">
      <string>Interval of action : Every</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="valueSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="correctionMode">
      <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="periodComboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>91</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
